bitnami/contour/values-production.yaml

```
@@ -59,6 +59,10 @@ configInline:
   leaderelection:
     # configmap-name: leader-elect
     configmap-namespace: '{{ .Release.Namespace }}'
+  ## This sets the service name that will be inspected for address details to be applied to Ingress objects.
+  # As by default the envoy services name deployed by this helm chart is 'contour-envoy', this configuration is needed
+  # in order to ensure, that the ingress status is automatically updated by contour.
+  envoy-service-name: contour-envoy
```

This is nice, but I think that the name of the Service can actually differ if, for instance, the property `.Values.fullnameOverride` is provided.

Instead, I think I would be great to use the exact same expression used to generate the name:
https://github.com/bitnami/charts/blob/6bf9dfd96083702704461d268895a1e6416fdde3/bitnami/contour/templates/envoy/service.yaml#L4-L6

```suggestion
  envoy-service-name: '{{ include "common.names.fullname" . }}-envoy'